tests/jballerina-unit-test/src/test/resources/test-src/record/closed_record_type_inclusion.bal

```diff
@@ -286,9 +286,21 @@ type SimpleConstNegateExpr record {|
     "-" op = "-";
 |};
 
+type R0 record {
+    int|string f = 1;
+};
+
+type R1 record {
+    *R0;
+    string f;
+};
+
+
 function testTypeInclusionWithFiniteField() {
     SimpleConstNegateExpr expr = {};
+    R1 r1 = { f : "hello"};
```

Comment:
Why are we adding this to `testTypeInclusionWithFiniteField` instead of introducing a new test?

Reply:
Move to separate test

Reply:
https://github.com/ballerina-platform/ballerina-lang/blob/bd04cbf4262fc1698cc8af2b248711df6c44ead6/tests/jballerina-unit-test/src/test/resources/test-src/record/open_record.bal#L688